sdk/samples/iot/provisioning/src/paho_iot_provisioning_example.c

```diff
@@ -461,8 +491,10 @@ static void sleep_for_seconds(uint32_t seconds)
 }
 
 // Print an az_span to the console
-static void print_az_span(az_span span)
+static void print_az_span(const char* str, az_span span)
```

Comment:
>print_az_span(const char* str, az_span span) [](start = 12, length = 44)

I liked the old function that prints the span. Now I am not sure what does 1st parameter do? Is this format like printf?

Reply:
yes. i was trying to find a way to consolidate all the lines like you asked.  This was an initial solution.